pkg/runtime/conversion_generation_test.go

```diff
@@ -143,7 +145,7 @@ func compareBuffers(t *testing.T, generatedFile string, existing, generated byte
 }
 
 func TestNoManualChangesToGenerateConversions(t *testing.T) {
-	versions := []string{"v1beta3"}
+	versions := []string{"v1beta3", "v1"}
```

Comment:
We do run each of these test for all values of testapi.
Right now testapi="v1beta1,v1beta3". We plan to include v1 there soon.
We will be running this for loop 6 times, when we do that :)